api/v1alpha1/mirrorpeer_types.go

```diff
@@ -63,7 +63,7 @@ type MirrorPeerSpec struct {
 
 	// SchedulingIntervals is a list of intervals at which mirroring snapshots are taken
 	// +kubebuilder:validation:Optional
-	SchedulingIntervals []string `json:"schedulingIntervals,omitempty"`
+	SchedulingIntervals []string `json:"schedulingIntervals,omitempty" deprecated:"true" deprecationMessage:"SchedulingIntervals has been deprecated and now only resides in DRPolicy. Any changes to this field will not affect the cluster state"`
```

Comment:
This won't show up in CRD. If you add this as a comment on Line 64 and regenerate CRD, it'll show up in CRD and users will see this deprecation message in UI.

Reply:
done